icu4c/source/i18n/cecal.cpp

```diff
@@ -76,7 +76,7 @@ CECalendar::operator=(const CECalendar& right)
 // Calendar framework
 //-------------------------------------------------------------------------
 
-int32_t
+int64_t
```

Comment:
> Are these all completely internal functions, or are some of them API or part of the subclassing interface?
yes, I only change handleComputeMonthStart is marked as @internal so it is not part of the API. 

> can't you still get integer overflow with int64_t?
Since the input are in type int32_t the computation result of these two int32_t should be still < the limitation of int64_t

> Are you doing anything to prevent that?
Let me think about this.